### PR TITLE
Bump kubekins

### PR DESCRIPTION
**Pull request by ixdy**
This is mostly to include #8005 from @Random-Liu.

/assign @fejta 
/hold

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -1231,7 +1231,7 @@ presubmits:
       preset-service-account: true
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180510-540e0e395-experimental
```
Comment (fejta) [inline prow/config.yaml]: What is experimental?

**Comment by fejta** [comment]
/lgtm
unhold at your leisure

File: prow/config.yaml (hunk 1 above)
Comment (ixdy) [reply prow/config.yaml]: it was added in https://github.com/kubernetes/test-infra/pull/6859. it seems to be used for canary jobs, but also in some real jobs?

Comment (ixdy) [reply prow/config.yaml]: cc @BenTheElder

**Comment by BenTheElder** [comment]
It's tools potentially newer than k/k master (IE test-infra mostly), while
the rest are pinned to either master or a release branch.

File: prow/config.yaml (hunk 1 above)
Comment (BenTheElder) [reply prow/config.yaml]: https://github.com/kubernetes/test-infra/blob/51cf02e75d8c5ed5da3dff8ea025b270f86de3d6/images/kubekins-e2e/Makefile#L24-L29

**Comment by fejta** [comment]
You are terrible at being OOO 😁

**Comment by ixdy** [comment]
ci-kubernetes-e2e-gce-canary looks green.

/hold cancel

**Pull request merged**